File: youlexuan_parent/youlexuan_user_interface/src/main/java/com/lazy/user/service/UserService.java
package com.lazy.user.service;

import java.util.List;
import com.lazy.pojo.TbUser;

import com.lazy.entity.PageResult;
/**
 * user服务层接口
 * @author lazy
 *
 */
public interface UserService {

	
	/**
	 * 增加
	*/
	public void add(TbUser user);

	/**
	 * 发送短信
	 * @param phone 当前页 码
	 * @return
	 */
    void sendMsg(String phone);

	boolean checkSmsCode(String phone, String code);
}

File: youlexuan_parent/youlexuan_user_web/src/main/webapp/js/service/userService.js
//user服务层
app.service('userService', function($http){
	// 保存、修改
	this.save = function(entity,code) {
		return $http.post('/user/add.do?code='+code, entity);
	}
	// 保存、修改
	this.sendMsg = function(phone) {
		return $http.post('/user/sendMsg.do?phone='+phone);
	}
});